lib/progress.c

```diff
@@ -369,100 +369,81 @@ void Curl_pgrsSetUploadSize(struct Curl_easy *data, curl_off_t size)
   }
 }
 
+/* returns the average speed */
+static curl_off_t trspeed(curl_off_t size, curl_off_t us)
+{
+  if(us < 1)
+    return size * 1000000;
```

Comment:
I would comment which unit/conversion factor this is.

Reply:
done!